server/methods/accounts/accounts.app-test.js

```diff
@@ -551,7 +553,7 @@ describe("Account Meteor method ", function () {
           email: fakeUser.emails[0].address,
           name: fakeUser.profile.addressBook[0].fullName
         })
-      ).to.not.throw(Meteor.Error, /Access denied/);
+      ).to.not.throw(Meteor.Error);
```

Comment:
this is not necessarily related to this change, but I coded an error by mistake and this test was still passing.
I find it good practice to not limit the _type_ of error thrown in a ".to.not.throw" test, so I changed it. lmk if you'd like to revert